src/y-webrtc.js

```diff
@@ -20,7 +20,9 @@ import * as awarenessProtocol from 'y-protocols/awareness'
 
 import * as cryptoutils from './crypto.js'
 
-const log = logging.createModuleLogger('y-webrtc')
+import { WebPubSubClient } from "@azure/web-pubsub-client";
+
+const log = logging.createModuleLogger('y-webrtc-azure-webpubsub')
```

Comment:
I copied this code from a local copy of the module that was renamed to `y-webrtc-azure-webpubsub`